generators/app/index.js

```diff
@@ -465,7 +469,20 @@ module.exports = class extends Generator {
       }
     });
     // Delete Poetry environment, because the developer will create its own Pyenv environment when installing the project
-    this.log("Deleting temporary virtual environment");
+    this.log(
+      `Deleting ${chalk.cyan.italic(
+        "temporary"
+      )} virtual environment ${chalk.cyan(this.answers.projectSlug)}`
+    );
+    this.log(
+      infoMessage(
+        `You will create a new ${chalk.bold.italic(
+          "pyenv"
+        )} environment during project installation (see generated README.md).`,
+        0,
+        false
+      )
+    );
```

Comment:
Could we move all this code into a unique function called `generateLockFile` ? I think it could be good to reduce the length of this file !

Reply:
Yes, there is a ticket in the roadmap to refactor this huge file :)